processing_belt_interpreter/RectangleStamped[] map_rects
recognition_objects_classifier/CircleObstacleStamped[] map_circles
recognition_objects_classifier/SegmentObstacleStamped[] map_segments

processing_belt_interpreter/RectangleStamped[] unknown_rects
recognition_objects_classifier/CircleObstacleStamped[] unknown_circles
recognition_objects_classifier/SegmentObstacleStamped[] unknown_segments

